libs/membership/api/src/lib/dashboard/subscriptions/dashboard-subscription.resolver.spec.ts

```diff
@@ -149,7 +152,8 @@ describe('DashboardSubscriptionResolver', () => {
             where: {email: 'foo@wepublish.ch'},
             create: {
               active: true,
-              email: 'foo@wepublish.ch',
+              email: `${generateRandomString()}@wepublish.ch`,
```

Comment:
I guess these changes come from trying to fix things?

Reply:
yeah probably

Reply:
not applicable anymore